src/grammar/grammar.ohm

```diff
@@ -143,6 +143,7 @@ Tact {
                     | ExpressionBinary "<<" ExpressionAdd --shl
                     | ExpressionBinary "&" ExpressionAdd --bin_and
                     | ExpressionBinary "|" ExpressionAdd --bin_or
+                    | ExpressionBinary "^" ExpressionAdd --bin_xor
```

Comment:
I believe there is a bug in this grammar: `^` has the same precedence as `&` and `^`. Incidentally, there is a bug with the rest of bitwise operators: e.g. `&` and `|` have the same precedence, and the shift operations too. See issue #263